src/mouse.c

```diff
@@ -137,17 +140,24 @@ void switch_screen(
     state->pointer_y = scale_y_coordinate(output->number, 1 - output->number, state);
 }
 
-void switch_desktop(device_t *state, output_t *output, int new_index, int direction) {
+#define MACOS_SWITCH_MOVE_X 10
```

Comment:
Those two lines seem to be redundant with line 20/21

Reply:
Whoops, good catch